www/js/app.js

```diff
@@ -358,7 +358,9 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         return false;
     });
     $('#btnTop').on('click', function() {
-        $("#articleContent").contents().scrollTop(0);
+        let articleContent = document.getElementById('articleContent');
+        console.log(articleContent.contentWindow);
```

Comment:
Be sure to remove logging before finalizing.

Reply:
Oh my bad, I shall remove it.